Entities/Common/Factory/ShopCommon.as

```diff
@@ -1,3 +1,5 @@
+funcdef void ShopMadeItem(CBitStream@);
```

Comment:
Could've just taken the params directly but w/e, can "just be refactored later" (lol)